app/config.py

```diff
@@ -144,6 +144,7 @@ class Config(object):
     # FEATURE FLAGS
     FF_SALESFORCE_CONTACT = env.bool("FF_SALESFORCE_CONTACT", True)
     FF_NEW_BRANDING = env.bool("FF_NEW_BRANDING", False)
+    FF_TEMPLATE_CATEGORY = env.bool("FF_NEW_BRANDING", False)
```

Comment:
Oof.. that would have caused some issues 🤦‍♂️
```suggestion
    FF_TEMPLATE_CATEGORY = env.bool("FF_TEMPLATE_CATEGORY", False)
```